frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_appearance/font_family_setting.dart

```diff
@@ -127,7 +133,7 @@ class _ThemeFontFamilySettingState extends State<ThemeFontFamilySetting> {
                     FlowySvgs.check_s,
                   )
                 : null,
-        onTap: () {
+        onTap: () async {
```

Comment:
why mark this async?
controller.close() does not return Future

Reply:
Hi...but DocumentAppearanceCubit::syncFontFamily does. I've missed the await on that line (I suggest to add the unawaited_futures lint rule).

Reply:
Thanks for clarifying.

Reply:
I don't think it quite makes sense to use async/await in an onTap handler.